app/views/layouts/_nav.html.erb

```diff
@@ -53,7 +53,7 @@
             <li><%= scp_link_to "<span class='fas fa-plus fa-fw'></span> Create a Study".html_safe, new_study_path, class: 'check-upload' %></li>
             <% if current_user.available_branding_groups.any? %>
               <li role="separator" class="divider"></li>
-              <li class="dropdown-header">My Brands</li>
+              <li class="dropdown-header">My Collections</li>
```

Comment:
Is this PR to include updating all user facing language to use 'Collections' over 'Brands'?

Reply:
yes, that's the goal

Reply:
There are still a few cases of "Branding Group" in `branding_groups_controller` that I think might need to be updated then

Reply:
ah, good find, I missed all the popup notifications.  fixing now